examples/post.v3.LR.historical.zppy_v3.cfg

```diff
@@ -0,0 +1,289 @@
+[ts]
+# 2 DEPRECATED PARAMETERS in zppy v3.0.0 (the new e3sm_to_cmip task renders these obsolete):
+# e3sm_to_cmip_environment_commands
+# ts_fmt (e.g., `ts_fmt = "cmip"`)
+active = True
+walltime = "00:50:00"
+years = "1985:2014:30"
+
+  [[ atm_monthly_180x360_aave ]]
+  # This subtask is a dependency for the e3sm_diags task's atm_monthly_180x360 and atm_monthly_180x360_aave_mvm subtasks.
+  # The following e3sm_diags sets require it:
+  # "enso_diags", "qbo", "area_mean_time_series"
+  # This subtask is also a dependency for the e3sm_to_cmip task's atm_monthly_180x360_aave subtask.
+  frequency = "monthly"
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = /home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+  vars = "FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW,U,ICEFRAC,LANDFRAC,OCNFRAC,PS,CLDICE,CLDLIQ,T,AODDUST,PSL,U10,QREFHT,LHFLX,FLDS,TGCLDIWP,TGCLDCWP,TMQ,FLNSC,FLUTC,FSDS,FSDSC,SOLIN,FSNSC,FSNTOAC,AODABS,AODVIS,AREL,TREFMNAV,TREFMXAV"
+  # Needed for mixed-phase partition:
+  #  vars = "LANDFRAC,CLDICE,CLDLIQ,T"
+
+  [[ land_monthly ]]
+  # This subtask is a dependency for the e3sm_to_cmip task's land_monthly subtask.
+  extra_vars = "landfrac"
+  frequency = "monthly"
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  mapping_file = map_r05_to_cmip6_180x360_aave.20231110.nc
+  vars = "FSH,RH2M,LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILICE,SOILLIQ,SOILWATER_10CM,TSA,TSOI,H2OSNO,TOTLITC,CWDC,SOIL1C,SOIL2C,SOIL3C,SOIL4C,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR"
+
+  [[ atm_daily_180x360_aave ]]
+  # This subtask is a dependency for the e3sm_diags task's atm_monthly_180x360 subtask.
+  # The following e3sm_diags sets require it:
+  # "tropical_subseasonal"
+  frequency = "daily"
+  input_files = "eam.h1"
+  input_subdir = "archive/atm/hist"
+  mapping_file = /home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc
+  # Needed for Wheeler Kiladis
+  vars = "FLUT,PRECT,U850"
+
+  [[ rof_monthly ]]
+  # The following e3sm_diags sets require it:
+  # "streamflow"
+  extra_vars = 'areatotal2'
+  frequency = "monthly"
+  input_files = "mosart.h0"
+  input_subdir = "archive/rof/hist"
+  mapping_file = ""
+  vars = "RIVER_DISCHARGE_OVER_LAND_LIQ"
+
+  [[ atm_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "eam.h0"
+  input_subdir = "archive/atm/hist"
+  mapping_file = "glb"
+
+  [[ lnd_monthly_glb ]]
+  # This subtask is a dependency for the global_time_series task.
+  frequency = "monthly"
+  input_files = "elm.h0"
+  input_subdir = "archive/lnd/hist"
+  mapping_file = "glb"
+  # vars = "FSH,RH2M,LAISHA,LAISUN,QINTR,QOVER,QRUNOFF,QSOIL,QVEGE,QVEGT,SOILWATER_10CM,TSA,H2OSNO,TOTLITC,CWDC,SOIL1C,SOIL2C,SOIL3C,SOIL4C,WOOD_HARVESTC,TOTVEGC,NBP,GPP,AR,HR"
+  vars = "" # This will tell zppy to use all available variables
```

Comment:
Consider to add a note about elapsed time for this 30 years run for this option, and caution about needing to adjust wall-time limit for longer simulations.